src/askui/models/ui_tars_ep/ui_tars_api.py

```diff
@@ -90,6 +90,11 @@ class UiTarsApiHandlerSettings(BaseSettings):
         min_length=1,
         validation_alias="TARS_API_KEY",
     )
+    tars_model_name: str = Field(
+        default="ui-tars",
```

Comment:
WDYT about keeping the default `"tgi"` instead of `"ui-tars"` so that we can omit the breaking change?

Reply:
If you think this change might break something, we can leave it as “tgi”.

However, the integration is already slightly broken (setting the TARS_URL and TARS_API_KEY variables didn’t work as documented), so at this stage I think we can change this variable freely and:

- use the most popular model's name (“ByteDance-Seed/UI-TARS-1.5-7B” - when using something like vLLM or LM Studio you'll see this in /v1/models)
- make the TARS_MODEL_NAME variable mandatory for the integration to run (I think this is the cleanest approach) 
- or simply leave “tgi”.

Frankly, I don’t understand why it’s named “tgi” by default :(